test/node.js

```diff
@@ -16,8 +17,32 @@ const fsstat = promisify(fs.stat)
 
 const IPFSRepo = require('../src')
 
+async function createTempRepo ({ init, dontOpen, opts }) {
+  const testRepoPath = path.join(__dirname, 'test-repo')
+  const date = Date.now().toString()
+  const repoPath = path.join(os.tmpdir(), 'test-repo-for-' + date)
+
+  const repo = new IPFSRepo(repoPath, opts)
+
+  if (init) {
+    await repo.init({})
+  } else {
+    await asyncNcp(testRepoPath, repoPath)
+  }
+
+  if (!dontOpen) {
```

Comment:
As far as I can tell `init`, `dontOpen` and `opts` are not used?

Reply:
Yeah, I think that I wanted to refactor the tests somehow using this function, but did not finished it 😅I will just remove it...